luigi_pipeline/lib/model/seqr_mt_schema.py

```diff
@@ -111,12 +97,6 @@ def xstart(self):
     def xstop(self):
         return variant_id.get_expr_for_xpos(self.mt.locus) + hl.len(variant_id.get_expr_for_ref_allele(self.mt)) - 1
 
-    @row_annotation()
-    def rg37_locus(self):
```

Comment:
I think we actually do want this for mito variants as well

Reply:
blocks approval

Reply:
Okay. I didn't add it back because the field `rg37_locus` wasn't annotated with the old code.